Source/Prism.Tests/Mvvm/AutoInitializeViewModelFixture.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Prism.Common;
+using Xunit;
+
+namespace Prism.Tests.Mvvm
+{
+    internal class MockParameters : ParametersBase
+    {
+        public MockParameters() : base() { }
+        public MockParameters(string query) : base(query) { }
+    }
+
+    internal enum MockEnum
+    {
+        None = 0,
+        Foo = 1,
+        Bar = 2,
+        Fizz = 3
+    }
```

Comment:
each type should be in its own file. Also mocks belong under the Mocks namespace